### ISSUE TITLE
Compatibility with GMDX, New Vision etc

### ISSUE THREAD

**Issue by ruuuuuuuuuuuuuuuuuuuaaaaaaaaa** [onnoj/DeusExEchelonRenderer#16]
Has this been tested with any of the standard QoL mods that most of us run? GMDX v9 being the obvious one to look at as it includes New Vision, HDTP etc.

**Comment by onnoj** [comment]
Nope. I'd love to have an overview with what works and what doesn't. I know that Revision doesn't work at all, and the renderer worked fine with one of the texture mods (but not sure which one it was).

That being said, I will only support the vanilla version, but will happily accept any merge requests that improve compatibility.

**Comment by LoadLineCalibration** [comment]
I've managed to run GMDX 9.0.3 with DeusExEchelonRenderer. There are some rendering issues, but it works. Also my custom maps are working just fine.

One need to add "localdata" to shortcut properties, or use different game exe.
![изображение](https://github.com/onnoj/DeusExEchelonRenderer/assets/44388228/5ce267b2-3d4e-4d44-9b12-242133996032)

**Comment by ruuuuuuuuuuuuuuuuuuuaaaaaaaaa** [comment]
> I've managed to run GMDX 9.0.3 with DeusExEchelonRenderer. There are some rendering issues, but it works. Also my custom maps are working just fine.
> 
> One need to add "localdata" to shortcut properties, or use different game exe. ![изображение](https://private-user-images.githubusercontent.com/44388228/329144763-5ce267b2-3d4e-4d44-9b12-242133996032.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzMTU1OTQsIm5iZiI6MTcxOTMxNTI5NCwicGF0aCI6Ii80NDM4ODIyOC8zMjkxNDQ3NjMtNWNlMjY3YjItM2Q0ZS00ZDQ0LTliMTItMjQyMTMzOTk2MDMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI1VDExMzQ1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmY2JjZDNiMGNlZmM0MDBiMjE2YWVmYzZjNDQ1ZGQwZDMxZjU0YTQxMTkwMTIyNmQwNzA2ZDQ5NjQ1NDk4MzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MyyoqwFVDdiankO6bYbcwmwlhVDrmQkvGD0VqpdWpew)

What does this localdata argument do?

**Comment by LoadLineCalibration** [comment]
> What does this localdata argument do?

Don't use user's directory for .ini files.

**Comment by IgnorantGenius** [comment]
> Nope. I'd love to have an overview with what works and what doesn't. I know that Revision doesn't work at all, and the renderer worked fine with one of the texture mods (but not sure which one it was).
> 
> That being said, I will only support the vanilla version, but will happily accept any merge requests that improve compatibility.

I just tried it on revision.  I think it will work if we can find the right tweaks.  I let it sit there at a white screen and eventually got a "precaching" message which appears when the game first loads up, so it could work if it is prepared for the mod, but I wouldn't know how to do that.  Here are some messages from the log.

[11:33:38.806] info:  No winproc detected, initiating bridge message channel for: 0000000000630EEC

This first line happened 49 times-

[11:37:56.029] warn:  Message channel UWM_REMIX_BRIDGE_REGISTER_THREADPROC_MSG handshake timeout. Retrying...

[11:37:57.189] info:  Message channel UWM_REMIX_BRIDGE_REGISTER_THREADPROC_MSG established.
[11:37:57.208] warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_DITHERENABLE
[11:38:12.632] info:  [RTX-Compatibility-Info] Trying to raytrace but not detecting a valid camera.

These "replacements" tell me that possibly renaming them correctly could get them to work with revision.

[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Ice.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Lens.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCubeCamera_A01_01.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCubeDigital_A02_01.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCubeIsogrid_A01_01.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCubeIsogrid_A02_01.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCubeIsogrid_A03_01.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_RTX_CompanionCube_A1_01.usd
[11:33:38.655] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_RollCage.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Healthpack.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Space.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Pillow_A.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Ceramic.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Wood.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCubeDigital_A01_01.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_SteamPunk_A01.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_ArtsAndCrafts.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Cubus.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_ArtsAndCrafts.usd
[11:33:38.656] warn:  [SecretReplacement] Could not find stage: rtx-remix\mods\DeusExEchelonRenderer\./SubUSDs/SM_Prop_CompanionCube_Cubus.usd

On another note, I wasn't able to get frame generation mods to work with Echelon.RTX-remix.  I tried the DLSS tweaks one as well as the other dll hook replacements that are out there.
